### PR TITLE
DOCSP-42303: Change username and password copy

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-42375>
Staging - <https://preview-mongodbmcmorisi.gatsbyjs.io/csharp/DOCSP-42303-username-password/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?

### CODE REVIEW

File: source/fundamentals/enterprise-authentication.txt [hunk 1]
```diff
@@ -219,8 +219,8 @@ using your directory-server username and password.
 The following examples specify the authentication mechanism using the following
 placeholders:
 
-- ``<username>``: Your LDAP username
-- ``<password>``: Your LDAP password
+- ``<db_username>``: Your LDAP username
+- ``<db_password>``: Your LDAP password
```
Comment (norareidy) [inline source/fundamentals/enterprise-authentication.txt:223]: Q: Similar question as above: are these different from the database username and password? If so, I'd remove the <db_> part of the placeholder and just keep as is. If not, I'd change to:
```suggestion
- ``<db_username>``: Your database username
- ``<db_password>``: Your database password
```

**Review by norareidy — CHANGES_REQUESTED** [review]
Left a couple questions!

File: source/fundamentals/enterprise-authentication.txt (hunk 1 above)
Comment (mcmorisi) [reply source/fundamentals/enterprise-authentication.txt:223]: Can revert both this and the other commit since it sounds like it's separate.

**Review by norareidy — APPROVED** [review]
lgtm!

**Pull request merged**